src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java

```diff
@@ -68,7 +69,8 @@ void testGetUnNotifiedHearingsWithStartDateTo() {
         assertNotNull(expected.getContent());
         assertEquals(2, expected.getContent().size());
         assertEquals(2, expected.getTotalElements());
-        assertEquals(expectedHearingIds, expected.getContent());
+        List<Long> actualHearingIds = expected.getContent().stream().sorted().collect(Collectors.toList());;
```

Comment:
Interesting. I resolved in another PR using new HashSet for both expected and actual Lists.

Reply:
https://build.hmcts.net/job/HMCTS_d_to_i/job/hmc-cft-hearing-service/job/PR-474/34/pipeline-console/?selected-node=86
idk, might still be broken but hasn't failed yet.